public/kbn_sankey_vis_controller.js

```diff
@@ -10,20 +11,24 @@ import { filterNodesAndLinks } from './lib/filter';
 const module = uiModules.get('kibana/kbn_sankey_vis', ['kibana']);
 let observeResize = require('./lib/observe_resize');
 
+const IS_DARK_THEME = chrome.getUiSettingsClient().get('theme:darkMode');
+
 module.controller('KbnSankeyVisController', function ($scope, $element, $rootScope, Private) {
   const sankeyAggResponse = Private(AggResponseProvider);
   $scope.emptyGraph = false;
 
   let svgRoot = $element[0];
   let resize = false;
-  let color = d3.scale.category10();
+  let color = d3.scale.category20();
```

Comment:
Hi @mmguero , can you explain please why changing this parameter?

Reply:
To give more variation to the colors, especially > 10 items ([see example](http://bl.ocks.org/aaizemberg/78bd3dade9593896a59d)).